src/stk/databases/mongo_db/value.py

```diff
@@ -168,7 +168,20 @@ def put(self, molecule, value):
         return self._put(HashableDict(json))
 
     def _put(self, json):
-        self._values.insert_one(json)
+        keys = dict(json)
+        keys.pop('v')
+
+        query = {'$or': []}
+        for key, value in keys.items():
+            query['$or'].append({key: value})
+
+        self._values.update_many(
```

Comment:
Just to check my understanding of pymongo, it will try to find database entries match any of the keys in the form {key: value}, then update the value with `json`. Will this update all the keys it matches in the filter? Or just a single one?

Reply:
The intended  behavior is as you described, and it should update all of them. So if you create an entry of the form 

```python

{    'v': 12
    'SMILES': 'C',
    'InChIKey': 'something',
}
```

And the database currently has two entries

```python

{ 'v': 10,
 'SMILES': 'C'
}

```

and 

```python

{ 'v': 21, 
'InChIKey': 'something'
}
```

You should end up with a database that has two entries, both of which are identical and both of which are 

```python
{    'v': 12
    'SMILES': 'C',
    'InChIKey': 'something',
}
```

Reply:
I think this is the most reasonable behavior and what most users would expect?